File: webflux-news/new-app/src/main/kotlin/com/urbaniak/krzysztof/webfluxnews/domain/Article.kt
package com.urbaniak.krzysztof.webfluxnews.domain

import java.time.Instant

data class Article(
    val sourceName: String,
    val author: String,
    val title: String,
    val description: String,
    val urlToArticle: String,
    val urlToImage: String,
    val publishedAt: Instant?
)

File: angular-news/src/app/shared/article/Article.ts
export class News {
  article: Article;

  constructor(article: Article) {
    this.article = article;
  }
}

export class Article {
    author: string;
    title: string;
    description: string;
    date: string;
    sourceName: string;
    articleUrl: string;
    imageUrl: string;


    constructor(
        author: string,
        title: string,
        description: string,
        date: string,
        sourceName: string,
        articleUrl: string,
        imageUrl: string
    ) {
        this.author = author;
        this.title = title;
        this.description = description;
        this.date = date;
        this.sourceName = sourceName;
        this.articleUrl = articleUrl;
        this.imageUrl = imageUrl;
    }
}
